src/discord/commands/guild/d.birthday.ts

```diff
@@ -131,6 +131,7 @@ export const dBirthday: SlashCommand = {
       .setName('get')
       .setDescription('Get someone\'s birthday!')
       .addUserOption(option => option
+        .setRequired(true)
```

Comment:
There's code down below that says "if the member isn't provided, default to the user who did the command"